packages/components/header/header.js

```diff
@@ -67,11 +68,24 @@ class Header {
    *
    */
   calculateBreakpoints() {
-    let childrenWidth = 0
-    for (let i = 0; i < this.navigationList.children.length; i++) {
-      childrenWidth += this.navigationList.children[i].offsetWidth
-      this.breakpoints[i] = childrenWidth
+    let right = 0
+
+    // Add navigation items on first load only
+    if (!this.navigationItems.length) {
```

Comment:
I'm not sure, but perhaps this block can be moved to its own method that gets called directly from the constructor, as it looks like it's just initialising the `navigationItems` array.